projects/ngx-youtube-player/src/lib/ngx-youtube-player.component.ts

```diff
@@ -69,9 +69,9 @@ export class YoutubePlayerComponent implements AfterContentInit {
 
   getProtocol() {
     const hasWindow = window && window.location;
-    const protocol = hasWindow
-      ? window.location.protocol.replace(':', '')
-      : 'http';
-    return protocol;
+    if (!hasWindow || window.location.protocol.startsWith('file')) {
```

Comment:
please use positive expression for the ``file``:
```js
const isFileProtocol = window.location.protocol.startsWith('file');
```
and revert back to ternary expression:
```js
const protocol = hasWindow && !isFileProtocol ? .... : ....;
```